scripting/include/tf_econ_data.inc

```diff
@@ -292,6 +292,12 @@ native ArrayList TF2Econ_GetParticleAttributeList(
  */
 native ArrayList TF2Econ_GetPaintKitDefinitionList();
 
+/**
+ * Return the definition index of the specific map.
+ * -1 if not valid.
+ */
+native int TF2Econ_GetMapDefinitionIndex(const char[] name);
```

Comment:
Intending to change the native name to `TF2Econ_TranslateMapNameToDefinitionIndex` to be more in line with the other natives that go between names / values.

I could go with `TF2Econ_GetMapDefinitionIndexFromName` or `TF2Econ_GetMapDefinitionIndexByName` as well; not particularly attached to any of the three names.  The last one would match what the game uses at least.

Reply:
i really have no opinion on the function name it can be whatever you want